numpy/core/src/_simd/_simd_inc_vector.h

```diff
@@ -0,0 +1,186 @@
+#if !NPY_SIMD
+    #error "Not a standalone header, only works through 'simd.dispatch.c.src'"
+#endif
+
+/************************************
+ ** Private Definitions
+ ************************************/
+// PySequenceMethods
+static Py_ssize_t
+simd__vector_length(PySIMDVectorObject *self)
+{
+    return simd_data_getinfo(self->dtype)->nlanes;
+}
+static PyObject *
+simd__vector_item(PySIMDVectorObject *self, Py_ssize_t i)
+{
+    const simd_data_info *info = simd_data_getinfo(self->dtype);
+    int nlanes = info->nlanes;
+    if (i >= nlanes) {
+        PyErr_SetString(PyExc_IndexError, "list index out of range");
+        return NULL;
+    }
+    npyv_lanetype_u8 *src = self->data + i * info->lane_size;
+    simd_data data;
+    memcpy(&data.u64, src, info->lane_size);
+    return simd_scalar_to_obj(data, info->to_scalar);
+}
+
+static PySequenceMethods simd__vector_as_sequence = {
+    (lenfunc) simd__vector_length,           /* sq_length */
+    (binaryfunc) NULL,                       /* sq_concat */
+    (ssizeargfunc) NULL,                     /* sq_repeat */
+    (ssizeargfunc) simd__vector_item,        /* sq_item */
+    (ssizessizeargfunc) NULL,                /* sq_slice */
+    (ssizeobjargproc) NULL,                  /* sq_ass_item */
+    (ssizessizeobjargproc) NULL,             /* sq_ass_slice */
+    (objobjproc) NULL,                       /* sq_contains */
+    (binaryfunc) NULL,                       /* sq_inplace_concat */
+    (ssizeargfunc) NULL,                     /* sq_inplace_repeat */
```

Comment:
Perhaps worth checking, but I think in other places we've switched to the C99 spelling,
```suggestion
    .sq_length = (lenfunc) simd__vector_length,
    .sq_item = (ssizeargfunc) simd__vector_item,
```

Reply:
my bad, that module was written long time ago and I forget to apply C99 on this part of code.